x/vault/keeper/keeper_test.go

```diff
@@ -132,20 +118,42 @@ func (s *KeeperTestSuite) CreateNewAsset(name, denom string, price uint64) uint6
 	}
 	s.Require().NotZero(assetID)
 
-	market := markettypes.Market{
-		Symbol:   name,
-		ScriptID: 12,
-		Rates:    price,
+	twa1 := markettypes.TimeWeightedAverage{
```

Comment:
please change this as well, for reference look liquidity module's change in the same PR.